packages/vuetify/src/components/VMenu/__tests__/VMenu.spec.ts

```diff
@@ -34,13 +34,15 @@ describe('VMenu.ts', () => {
         value: false,
         fullWidth: true
       },
+      scopedSlots: {
+        activator: '<button v-on="props.on"></button>'
```

Comment:
Don't these need to be functions?

Reply:
They don't have to be https://vue-test-utils.vuejs.org/api/options.html#scopedslots

Reply:
@KaelWD It can be a fn or a string template.